OpenRA.Game/Network/Session.cs

```diff
@@ -187,6 +188,7 @@ public class Global
 			public int StartingCash = 5000;
 			public string TechLevel = "none";
 			public string StartingUnitsClass = "none";
+			public string GameSpeedType = "default";
```

Comment:
What I mean is that this default type is hard-coded to a specific undocumented ID.